### ISSUE TITLE
More a request than an issue - Course shortname link in breadcrumbs on course home page

### ISSUE THREAD

**Issue by andrewrowatt-masseyuni** [blackboard-open-source/moodle-theme_snap#133]
Hi @gthomas2 & team,

We went live on November 15 with Moodle 3.5 and the latest version of Snap we could get our hands on.  Great work yet again on this theme.  

One change (from 3.1) that our staff have noticed is that the course shortname is not displayed in the breadcrumbs on the course home page.  Is this an issue for many staff as they teach into a number of different offerings which have the same fullname but unique shortname.  Worse case is a staff member with 25 courses.  A more typical example would be 3 or 4 per year (we have 33,000 courses).  

This is no problem for students (and actually a good change) but for staff it is another matter.

Anyway, long story short - are there plans to add the shortname link back to the breadcrumbs or make the change an option?  See screen-shot for example - the "192022_2018_S2B2" part is what we want back.

Any steer appreciated.

Thanks,
   Andrew

![image](https://user-images.githubusercontent.com/5333879/48729978-e91fdc00-ec9d-11e8-96f0-294409df6d98.png)

**Comment by gthomas2** [comment]
Hi Andrew

The shortname was removed intentionally from the breadcrumbs as far back as 3.3. Since the course title is shown on the course home page it didn't make sense to include the shortname in the breadcrumb. This decision was made by an expert in UX (not me). I can see how for some institutions this might be less than ideal but so far this is the first complaint I'm aware of.

If you modify the following file around line 1777:

theme/snap/classes/output/core_renderer.php

FROM

            // Remove link to home/dashboard as site name/logo provides the same link.
            if ($item->key === 'home' || $item->key === 'myhome' || $item->key === 'dashboard') {
                continue;
            }

TO

            // Remove link to home/dashboard as site name/logo provides the same link.
            if ($item->key === 'home' || $item->key === 'myhome' || $item->key === 'dashboard') {
                //continue;
            }

Then you should once again get the course shortname in the breadcrumb bar. I've not tested this - this is just from quickly looking at the code.

**Comment by andrewrowatt-masseyuni** [comment]
Thanks @gthomas2 that has been most helpful.  Just to reiterate - not a complaint at all as the change is good for the majority.   I will pass this onto our dev team.
